atomic_reactor/plugins/prepub_flatpak_create_oci.py

```diff
@@ -124,10 +138,22 @@ def run(self):
 
         ref_name, outfile, tarred_outfile = self.builder.build_container(tarred_filesystem)
 
+        self.log.info('Marking filesystem image "%s" for removal', self.workflow.builder.image_id)
+        defer_removal(self.workflow, self.workflow.builder.image_id)
+
         image_id = self._get_oci_image_id(outfile)
         self.log.info('New OCI image ID is %s', image_id)
         self.workflow.builder.image_id = image_id
 
+        labels = Labels(df_labels)
+        _, image_name = labels.get_name_and_value(Labels.LABEL_TYPE_NAME)
+        _, image_version = labels.get_name_and_value(Labels.LABEL_TYPE_VERSION)
+        _, image_release = labels.get_name_and_value(Labels.LABEL_TYPE_RELEASE)
+
+        name = '{}-{}'.format(self.key, image_name)
+        tag = '{}-{}'.format(image_version, image_release)
+        self._copy_oci_to_local_storage(outfile, name, tag)
```

Comment:
It would be nice to have a comment explaining why we don't need to also `defer_removal` of the image in docker storage.

Reply:
Done. I also added a docstring to the new method